.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2.1
 
 orbs:
   cli: circleci/circleci-cli@0.1
-  orb-tools: circleci/orb-tools@9.0
+  orb-tools: circleci/orb-tools@dev:orb-pack
```

Comment:
This is temporary so I can fully test the commands, since some are (at least very difficult) to test on their own. Although not impossible - looking into it on second thought.